src/PlacesAutocomplete.js

```diff
@@ -49,7 +49,13 @@ class PlacesAutocomplete extends React.Component {
   }
 
   autocompleteCallback(predictions, status) {
-    if (status != this.autocompleteOK) { console.error('place autocomplete failed'); return; }
+    if (status != this.autocompleteOK) {
+      this.props.onError()
+      if (this.props.clearItemsOnError) {
+        this.setState({ autocompleteItems: [] })
```

Comment:
Also now that I'm thinking more about error, do you think passing `status` to `onError` callback would be useful to the user of the library? @cburbank, would love to hear your thoughts on this 😄